mkdocs.yml

```diff
@@ -83,6 +84,7 @@ nav:
     - Installation: Software/Installation.md
     - Third-party software: Software/Third-party.md
     - Interacting with Clinica: Software/InteractingWithClinica.md
+    - Contributing to the Github repository : Software/CONTRIBUTING.md
```

Comment:
It totally makes sense to have this section here.
The usual trick to also satisfy what I wrote in my first comment is to link or include the content of the `CONTRIBUTING.md` file at the root. Let me know if this isn't clear.

Reply:
I am not sure this is clear but I am testing a way to keep it where it should to follow the convention and add the content as a snippet there !

Reply:
I did not know of this solution, thanks :) !!

Reply:
From what I have tested : 
- Mkdocs does not like symlinks outside its base repository (docs) : it's the current commit which can't be built apparently
- I tried to use snippets but since we set the base_path at docs too we cannot use it either

I could use a hard link like https://github.com/aramis-lab/clinica/blob/dev/README.md but you might not like this since it would be hardcoded with the dev version. WDYT ? @NicolasGensollen